### ISSUE TITLE
Feature request: parallel package building

### ISSUE THREAD

**Issue by RossComputerGuy** [void-linux/void-packages#28711]
I've noticed `xbps-src` has no parallel package building. I think it would be beneficial to support a method to build packages in parallel instead of sequentially. This with parallel compiling and distcc could bring compile times down by a lot on large packages when multiple packages are rebuilt. Maybe a `-pJ` flag (package-jobs) would set how many packages to build at once. Since `xbps-src` is written as a shell script, there's most likely a way to figure this out.

**Comment by ericonr** [comment]
Sticking this inside xbps-src doesn't really make sense to me. There's a lot of complexity and decisions to take on how to build packages in parallel: overlays with the same masterdir or multiple masterdirs? How to control/monitor all the jobs? And others.

Imo, writing something that can drive xbps-src is more feasible and more flexible. Could be an extension of xbulk or maybe (and probably better) something based on what already runs in the official builders.

**Comment by RossComputerGuy** [comment]
I think implementing this into xbulk would be the best. Then the flag can be `-j`.

**Comment by the-maldridge** [comment]
xbulk can do this today, and other projects are focused on doing this on distributed fleets.  Closing as out of scope but feel free to continue the discussion.

**Comment by RossComputerGuy** [comment]
I couldn't find other projects that does this with `xbps-src`, I don't know if they just don't exist or DuckDuckGo is having a hard time finding them.
